src/components/main/NoteContainer.js

```diff
@@ -58,15 +66,16 @@ const NoteContainer = () => {
     isError: isAppDataError,
   } = useAppData();
 
+  const errorDataMsg = t('Error getting data.');
   useEffect(() => {
     if (isAppDataError) {
-      showErrorToast(t('Error getting data.'));
+      showErrorToast(errorDataMsg);
```

Comment:
I changed this because the linter asks me otherwise to add `t` as the list of dep in the useEffect.

`warning  React Hook useEffect has a missing dependency: 't'. Either include it or remove the dependency array. Outer scope values like 'errorDataMsg' aren't valid dependencies because mutating them doesn't re-render the component  react-hooks/exhaustive-deps`